ui/tests/unit/models/variable-test.js

```diff
@@ -48,4 +48,68 @@ module('Unit | Model | variable', function (hooks) {
     assert.equal(Object.entries(model.items)[1][0], 'myVar');
     assert.equal(Object.entries(model.items)[1][1], 'myValue');
   });
+
+  test('it computes linked entities', function (assert) {
+    let store = this.owner.lookup('service:store');
+
+    let model = store.createRecord('variable');
+    model.setProperties({
+      path: 'jobs/my-job-name/my-group-name/my-task-name',
```

Comment:
`question`:  How will this handle parameterized and periodic jobs?

Reply:
linking should work the same way; jobs.job and jobs.job.task-group. I won’t be linking to the dispatched instance of it, but that would be less useful anyhow. The nature of Secure Vars means that any dispatched job that matches the glob pattern of the variable has access (not a specific dispatch)